http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/RuntimeServiceInterp.scala

```diff
@@ -237,7 +239,7 @@ class RuntimeServiceInterp[F[_]: Parallel](config: RuntimeServiceConfig,
       _ <- if (runtime.asyncRuntimeFields.isDefined) {
         clusterQuery.updateClusterStatus(runtime.id, RuntimeStatus.PreDeleting, ctx.now).transaction >> publisherQueue
           .enqueue1(
-            DeleteRuntimeMessage(runtime.id, Some(ctx.traceId))
+            DeleteRuntimeMessage(runtime.id, req.deleteDisk, Some(ctx.traceId))
```

Comment:
I thought we were going to detach the disk in front Leo? Or is that a separate PR?

Reply:
yea, that'll be a separate PR. This is just to support `deleteDisk` flag